code/modules/chemistry/Reagents-FoodDrink.dm

```diff
@@ -4395,7 +4395,7 @@ datum
 				return
 
 		fooddrink/alcoholic/lingtea
-			name = "Ling Island Iced Tea"
+			name = "ling island iced tea"
```

Comment:
```suggestion
			name = "Ling Island iced tea"
```
Not an actual place but for the sake of consistency

Reply:
theta 👀👀👀👀